public/js/icinga/behavior/modal.js

```diff
@@ -187,6 +199,8 @@
      * @param $modal {jQuery} The modal element
      */
     Modal.prototype.hide = function($modal) {
+        modalOpener.removeAttribute('style');
```

Comment:
This removes the entire style attribute. You should only reset/remove what you've added. (i.e. `pointerEvents`)